src/Linter/ProcessLinter.php

```diff
@@ -160,10 +160,6 @@ private function prepareCommand($path)
     {
         $executable = ProcessUtils::escapeArgument($this->executable);
 
-        if (defined('HHVM_VERSION')) {
-            $executable .= ' --php';
-        }
-
         $path = ProcessUtils::escapeArgument($path);
 
         return sprintf('%s -l %s', $executable, $path);
```

Comment:
I would move `ProcessUtils::escapeArgument()` calls here instead of using variables.